admin-console/admin-console.md

```diff
@@ -0,0 +1,100 @@
+## OpenShift 4.4+ Designs
+Access design documentation for features slotted in the 4.4 release of OpenShift. Click links below for specific section details.
+
+- [Bulk actions](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/bulk-actions/bulk-actions)
+- [Cron Jobs](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/cron-jobs/cron-jobs)
+- [Diagnostics](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/diagnostics/diagnostics)
+- [Favoriting](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/favoriting/favoriting)
+- [Health Checks](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/health-checks/health-checks)
+- [Search page](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/search/search)
+- [Toolbars](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/toolbars/toolbars)
+
+## OpenShift 4.3 Designs
+Access design documentation for features slotted in the 4.3 release of OpenShift. Click links below for specific section details.
+
+- [Alertmanager config](http://openshift.github.io/openshift-origin-design/admin-console/monitoring/alertmanager-config/alertmanager-config)
+- [Chargeback](http://openshift.github.io/openshift-origin-design/future-openshift/admin-console/chargeback/chargeback)
+- [Custom Catalogs](http://openshift.github.io/openshift-origin-design/admin-console/future-openshift/custom-catalogs/custom-catalogs)
+- [Dashboards](http://openshift.github.io/openshift-origin-design/admin-console/dashboards/4.3)
```

Comment:
@matthewcarleton This link is currently broken so what you copied here won't work either. Can you update this link to the following: http://openshift.github.io/openshift-origin-design/admin-console/dashboards/4.3/4.3-dashboards

FYI @andybraren

Reply:
nice catch!